services/QuillLMS/app/controllers/teachers/units_controller.rb

```diff
@@ -8,19 +8,14 @@ class Teachers::UnitsController < ApplicationController
   before_action :authorize!
 
   def create
-    units_with_same_name = current_user.units_with_same_name(params[:unit][:name])
```

Comment:
This is such a crazy line of code when I think about it.  I'm glad you've killed it.